Composer/packages/server/src/models/bot/luPublisher.ts

```diff
@@ -218,7 +222,7 @@ export class LuPublisher {
   private _loadLuConatents = async (paths: string[]) => {
     return await this.builder.loadContents(
       paths,
-      this.config?.defaultLanguage || '',
+      'en-us',
```

Comment:
Why has this change been made? Seems unrelated to the PR.

Reply:
@lei9444 we'll take this change but please follow-up here as it seems to hardcode 'en-us'

Reply:
Sorry, @a-b-r-o-w-n @cwhitten this is another issue @luhan2017 ever mentioned. And it is not a good solution to use hardcode here. I commit another PR to fix this #3281